### PR TITLE
Improved the responsive UI for ResourceBoardView and Shifting boardView

### LINKED ISSUE #2028: Improvement in the mobile view
Current Shifting navbar in Mobile-view
image

Improvement in the shifting navbar
image

### PR DESCRIPTION
**Pull request by kunatastic**
Fixes #2028 

I have updated the navbar for shifting & resources page

The updated new mobile view:

### CODE REVIEW

**Comment by gigincg** [comment]
@kunatastic Can you increase the Gap and keep the filters and the List View button in the same row. And I believe you've kept the after ss twice in the PR. Would be nice if you write some description in the PR text for easy review.

Nice find and Nice work

**Comment by kunatastic** [comment]
Sure!! I will make sure to add the description from next time.

**Comment by nihal467** [comment]
tested

**Comment by gigincg** [comment]
@nihal467 Removing test cleared, I've requested to keep The 2 buttons in the same row. @kunatastic mentioned he'll be incorporating those too.

**Comment by kunatastic** [comment]
The search bar is a component whose property was sticky to top-0 so I didn't change it. 
The updated new mobile view:

File: src/Components/Shifting/BoardView.tsx [hunk 1]
```diff
@@ -85,66 +85,69 @@ export default function BoardView() {
...
+            componentRight={
+              <GetAppIcon
+                className="cursor-pointer mt-2 ml-2"
+                onClick={triggerDownload}
+              />
+            }
+            breadcrumbs={false}
+          />
+        </div>
+        <div className="flex items-end  md:items-center justify-between md:w-2/3 flex-col md:flex-row">
```
Comment (khavinshankar) [inline src/Components/Shifting/BoardView.tsx]:
```suggestion
        <div className="flex items-end pt-2 space-y-2 md:items-center justify-between md:w-2/3 flex-col md:flex-row">
```

File: src/Components/Resource/ResourceBoardView.tsx [hunk 2]
```diff
@@ -85,57 +85,62 @@ export default function BoardView() {
...
-        </button>
-        <div className="flex items-start gap-2">
-          <button
-            className="flex leading-none border-2 border-gray-200 bg-white rounded-full items-center transition-colors duration-300 ease-in focus:outline-none hover:text-primary-600 focus:text-primary-600 focus:border-gray-400 hover:border-gray-400 rounded-r-full px-4 py-2 text-sm"
-            onClick={(_) => setShowFilters((show) => !show)}
-          >
-            <i className="fa fa-filter mr-1" aria-hidden="true"></i>
-            <span>Filters</span>
-          </button>
+        <div className="flex items-end  md:items-center justify-between md:w-2/3 flex-col md:flex-row">
```
Comment (khavinshankar) [inline src/Components/Resource/ResourceBoardView.tsx]:
```suggestion
        <div className="flex items-end pt-2 space-y-2 md:items-center justify-between md:w-2/3 flex-col md:flex-row">
```

**Review by khavinshankar — COMMENTED** [review]
some padding-top to the search bar and some space between all components would make it better.

the UI might look something like this,
image

**Comment by gigincg** [comment]
image

@kunatastic In some screen sizes (Tablet Portrait Mode) The Nav area looks broken. The filter button is half off screen and the search bar is too small. Can you fix ?

Suggestion: Keep Search bar in a separate row and the rest in a new row for these screens.

Also, the Resource screen is practically a clone of the Shifting Screen in terms of design, so you could possibly pass on the fixes to that screen as well. This could be a separate Issue/Pull Request

**Review by gigincg — CHANGES_REQUESTED** [review]
UI Breaking in iPad/Tablet Portrait View

**Comment by kunatastic** [comment]
Sure I will fix that.

**Comment by kunatastic** [comment]
Updated Tablet view 

image

**Comment by nihal467** [comment]
pwkefewop

@kunatastic @gigincg all the buttons like list view, patient name, etc.. seem really close and each of them feels like having different size

**Pull request closed without merging**